app/utils/polyfill.ts

```diff
@@ -0,0 +1,10 @@
+import { Platform } from "react-native"
+
+// Polyfill for Intl.NumberFormat. 
+// See: https://stackoverflow.com/a/55861894
+if(Platform.OS === 'android') { // only android needs polyfill
+  // eslint-disable-next-line import/no-unassigned-import
```

Comment:
@bodymindarts @samerbuna Is there another syntax for this import eslint will be happy with? Otherwise can we remove [this](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unassigned-import.md) rule?

Reply:
I'd rather keep the rule (which is good for our own code) and specify an `allow` list for things we don't have control over.